README.md

```diff
@@ -87,6 +87,26 @@ One.getTid();
 ```
 * See example of usage [here](https://github.com/thunderheadone/one-sdk-react-native/tree/master/example/src/App.tsx#L78)
 
+### Opt an end-user out of tracking
+To opt an end-user out of all tracking options, when the end-user does not give permission to be tracked in the client app, call the following method:
```

Comment:
Please add a file name where it needs to be called.

Reply:
Customers can call it anywhere it fits their use case.  There is not a standard place to put it.